monai/apps/pathology/transforms/post/array.py

```diff
@@ -0,0 +1,271 @@
+    """
+    generate mask used in `watershed`. Only points at which mask == True will be labeled.
+
+    Args:
+        softmax: if True, apply a softmax function to the prediction.
+        sigmoid: if True, apply a sigmoid function to the prediction.
+        threshold: if not None, threshold the float values to int number 0 or 1 with specified theashold.
+        argmax: if True, execute argmax function on input data.
+        remove_small_objects: whether need to remove some objects in the marker. Defaults to True.
+        min_size: objects smaller than this size are removed if `remove_small_objects` is True. Defaults to 10.
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __init__(
+        self,
+        softmax: bool = False,
+        sigmoid: bool = False,
+        threshold: Optional[float] = None,
+        argmax: bool = False,
+        remove_small_objects: bool = True,
+        min_size: int = 10
+    ) -> None:
+        self.activations = Activations(sigmoid=sigmoid, softmax=softmax)
+        self.asdiscrete = AsDiscrete(threshold=threshold, argmax=argmax)
+        if remove_small_objects:
+            self.remove_small_objects = RemoveSmallObjects(min_size=min_size)
+        else:
+            self.remove_small_objects = None  # type: ignore
+
+    def __call__(self, prob_map: NdarrayOrTensor) -> NdarrayOrTensor:
+        """
+        Args:
+            prob_map: probability map of segmentation, shape must be [C, H, W]
+        """
+        prob_map = convert_to_tensor(prob_map, track_meta=get_track_meta())
+
+        pred = self.activations(prob_map)
+        pred = self.asdiscrete(pred)
+
+        pred = label(pred)[0]
+        if self.remove_small_objects:
+            pred = self.remove_small_objects(pred)
+        pred[pred > 0] = 1
+
+        return convert_to_dst_type(pred, prob_map, dtype=np.uint8)[0]
+
+
+class GenerateProbabilityMap(Transform):
+    """
+    Generate foreground probability map by hover map. The more parts of the image that cannot be identified as foreground areas,
+    the larger the grey scale value. The grey value of the instance's border will be larger.
+
+    Args:
+        kernel_size: the size of the Sobel kernel. Defaults to 21.
+        remove_small_objects: whether need to remove some objects in segmentation results. Defaults to True.
+        min_size: objects smaller than this size are removed if `remove_small_objects` is True. Defaults to 10.
+
+    """
+
+    backend = [TransformBackends.NUMPY]
+
+    def __init__(
+        self,
+        kernel_size: int = 21,
+        min_size: int = 10,
+        remove_small_objects: bool = True,
+    ) -> None:
+        self.sobel_gradient =  SobelGradients(kernel_size=kernel_size)
+        if remove_small_objects:
+            self.remove_small_objects = RemoveSmallObjects(min_size=min_size)
+        else:
+            self.remove_small_objects = None  # type: ignore
+
+    def __call__(self, mask, hover_map):
```

Comment:
Could you please help add the typehint for args?

Thanks in advance.

Reply:
Have added, thanks!